lib/common/imageTool.py

```diff
@@ -24,7 +25,7 @@ def dump_result_to_json(self, data, output_fp):
         with open(output_fp, "wb") as fh:
             json.dump(data, fh)
 
-    def convert_video_to_images(self, input_video_fp, output_image_dir_path, output_image_name=None, exec_timestamp_list=[]):
+    def convert_video_to_images(self, input_video_fp, output_image_dir_path, output_image_name=None, exec_timestamp_list=[], comp_mode=0):
```

Comment:
comp_mode be True/False is fine to me currently; using default 0 just for consideration of future extension